video/out/mac/common.swift

```diff
@@ -42,12 +42,15 @@ class Common: NSObject {
 
     var cursorVisibilityWanted: Bool = true
 
+    var colorSpace: NSColorSpace?
```

Comment:
i believe this variable is not needed.

Reply:
It's used in `updateICCProfile` for `setRenderICCProfile` in `cocoa_cb_common.swift`.

Reply:
what i meant here, is that we don't need a variable that is available to the whole object of that class but just a function local variable would have done perfectly fine.